src/test/java/uk/gov/companieshouse/web/accounts/service/company/impl/CompanyServiceImplTests.java

```diff
@@ -33,24 +35,33 @@
     @Mock
     private CompanyResourceHandler companyResourceHandler;
 
+    @Mock
+    private CompanyGet companyGet;
+
     @InjectMocks
     private CompanyService companyService = new CompanyServiceImpl();
 
     private static final String COMPANY_NUMBER = "companyNumber";
 
+    private static final String COMPANY_URI = "/company/" + COMPANY_NUMBER;
+
+    private static final String INVALID_COMPANY_URI = COMPANY_URI + "/blah";
```

Comment:
Humour aside—and it did give me a chuckle—we should be aiming to keep the codebase as professional as possible, perhaps `invalid-company-uri` or similar would work better here rather than `blah`.